### DISCUSSION TITLE
download bundle using rauc 1.6

### DISCUSSION THREAD

**Discussion by nadhem09** [rauc/rauc#1010] [Q&A]
hi all,
due to our project constraint we are using rauc 1.6 version (that does not support streaming) 
i'm asking if there is a built in  solution to download this bundle using rauc 1.6

**Comment by jluebbe** [comment]
RAUC 1.6 can only download small bundles, especially for use with casync. Without streaming, the normal approach is to download the bundle before calling RAUC.

RAUC 1.8 should not need newer dependencies, so it should be a trivial update. Which constraint keeps you on 1.6?

**Reply by nadhem09** [reply]
@jluebbe thanks for your fast reactivity
first  to respond to your question: the yocto distribution is managed by another team and migration for later versions is not planned yet.
i have some other few questions :
when you said small bundle how much size approximately talking about ? 
does casync is compatible with NAND flash?

**Reply by ejoerns** [reply]
You can update single recipes within your layers. And we have backported RAUC 1.8 to all yocto LTS releases.
I don't see a reason here to base new designs on legacy software.

**Reply by nadhem09** [reply]
yeah i know but it's not just a technical problem. but also team priority and capacity . so  we must work with 1.6  :(
so  can you give me a confirmation about Max bundle size and compatibility with NAND flash ?

**Reply by jluebbe** [reply]
casync is not compatible with NAND without patches, so if you can't update, I'd suggest downloading the bundle before starting the installation outside of RAUC.

**Reply by ejoerns** [reply]
> yeah i know but it's not just a technical problem. but also team priority and capacity . so we must work with 1.6 :(

Building workarounds that would not be required if someone bumped a single package sounds like more waste of capacity to me :wink: 

> so can you give me a confirmation about Max bundle size and compatibility with NAND flash ?

It's 8MiB (so just for a bundle that contains casync index files but no images).
NAND is not supported as casync never accepted the required patches: https://github.com/systemd/casync/pull/227
Without them applied it would make no sense to enable the update handler within RAUC (also refert to existing issue https://github.com/rauc/rauc/issues/715)

**Reply by nadhem09** [reply]
thank you so much for your time guys .
